_posts/2011-04-13-odata-service-validation-tool.html

```diff
@@ -33,5 +33,5 @@
 <p><a href="/assets/screenshot3.png"><img class="alignnone size-full wp-image-342" alt="screenshot3" src="/assets/screenshot3.png" width="475" height="612" /></a></p>
 <p>The current (alpha) release of this tool runs a small representative set of rules derived from the specification. You can see a current list of rules <a href="https://services.odata.org/validation/roadmap.htm#rules">here</a>.</p>
 <p>We are actively working on adding more rules to increase the coverage across the specification so that compliance can be determined with a much more exhaustive set of rules. The goal is to add new rules to the service on a regular cadence.</p>
```

Comment:
Maybe outside the scope of the PR, but got this error when I visited https://services.odata.org/validation/roadmap.htm

Also it's another URL with a subdomain.

![image](https://github.com/OData/odataorg.github.io/assets/8460169/b8a972fc-684d-4645-93e9-41c44ea0c0f6)

Reply:
@habbes This is a dead link. I came across it and opted to leave it as it is. https://services.odata.org is a valid link though but the additional segments make the link invalid. There was no sense in converting it into relative URL if that too would result into a 404.